browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -2014,7 +2014,7 @@ Are you sure you want to do this?
       Balance:
     </message>
     <message name="IDS_WALLET_BUY_MAINNET_BUTTON_TEXT" desc="Buy button on main net text">
-      Continue to Wyre
+      Continue
```

Comment:
I don't think we need to touch that, or maybe just remove string at all or just leave for now as it's not visible. The thing is when we change it, it will be triggered to re-translate to all languages, but we don't need that, because we are going to change it after to that to `.. to Ramp` and will trigger re-translation again

Reply:
done!